tests/integration/dbapi/async/test_errors_async.py

```diff
@@ -92,7 +92,8 @@ async def test_database_not_exists(
             await connection.cursor().execute("show tables")
 
     assert (
-        str(exc_info.value) == f"Database {new_db_name} does not exist"
+        str(exc_info.value)
+        == f"Engine {engine_name} is not attached to {new_db_name}, but to {database_name}"
```

Comment:
IMHO message like "Engine {engine_name} is  attached to {database_name} instead  of {new_db_name}" is clearer.

Reply:
Sounds good